index.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+# Override the github.* context env vars with the ones that we got from the user
+# that may (or may not be) different from the original $GITHUB_* env vars.
+export GITHUB_TOKEN="$INPUT_TOKEN"
+export GITHUB_SERVER_URL="$INPUT_GITHUB_SERVER_URL"
+export GITHUB_REPOSITORY="$INPUT_REPOSITORY"
+
+# This is the default host that gh uses for clones and commands without a repo
+# context (a .git folder). We use Bash string magic to get the github.com part
+# from a full origin (no pathname) like https://github.com -> github.com
+export GH_HOST="${GITHUB_SERVER_URL#*//}"
+
+echo '🟪 Configuring local Git using gh auth setup-git...'
+gh auth setup-git
+
+# https://stackoverflow.com/a/28180781
+echo "🟪 Cloning $GITHUB_REPOSITORY.wiki into $INPUT_PATH..."
+cd "$INPUT_PATH"
+gh repo clone "$GITHUB_REPOSITORY.wiki" .git -- --bare
```

Comment:
Just realized that this could have a potential conflict with the documented usage to use `path: .`, since a `.git` directory will likely already exist at `./`.

Reply:
ohh shoot yeah
thats a problem when just plain `git init`-ing or `git clone --bare`-ing 😱

Reply:
we could use:
- GIT_DIR is the location of the .git folder. If this isn’t specified, Git walks up the directory tree until it gets to ~ or /, looking for a .git directory at every step.
- GIT_WORK_TREE is the location of the root of the working directory for a non-bare repository. If --git-dir or GIT_DIR is specified but none of --work-tree, GIT_WORK_TREE or core.worktree is specified, the current working directory is regarded as the top level of your working tree.
from https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables

and basically do
```sh
export GIT_DIR=../wiki.git
export GIT_WORK_TREE="$INPUT_PATH"
gh repo clone "$GITHUB_REPOSITORY" "$GIT_DIR" -- --bare
git config --unset core.bare
git reset
```
?? 🤔